tests/ui_tests/test_login_page.py

```diff
@@ -0,0 +1,55 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# pylint: disable=W0621
+import allure
+import pytest
+
+from tests.ui_tests.app.page.admin_intro.admin_intro_page import AdminIntroPage
+from tests.ui_tests.app.page.login.login_page import LoginPage
+
+
+def test_check_login_to_adcm(app_fs, adcm_credentials):
+    login_page = LoginPage(app_fs.driver, app_fs.adcm.url).open()
+    login_page.check_all_elements()
+    login_page.login_user(**adcm_credentials)
+    with allure.step("Check if user has been authorized"):
+        intro_page = AdminIntroPage(app_fs.driver, app_fs.adcm.url)
+        login_page.wait_url_contains_path(intro_page.path)
+        assert intro_page.path in app_fs.driver.current_url, \
+            f"Page '{intro_page.path}' has not been opened"
+        intro_page.header.check_auth_page_elements()
+
+
+@pytest.mark.parametrize("name, password", [("admin", " "), (" ", "admin")],
```

Comment:
No nothing?) It looks like we should be unable to press login if one of the fields is empty. Ask the web team about it.
Also need to add invalid user and invalid password cases

Reply:
added two more tests with empty name and pass values and as a result - the unavailable login button

Reply:
Ok. My bad. I didn't notice a space between quotes. Consider changing ids and it will be ok!